mscore/inspector/inspector.cpp

```diff
@@ -570,6 +570,9 @@ InspectorArticulation::InspectorArticulation(QWidget* parent)
       {
       ar.setupUi(addWidget());
 
+      if (toArticulation(inspector->element())->isOrnament())
+            ar.title->setText("Ornament");
```

Comment:
What about when there is a list selection containing ornaments and articulations?
There are other Inspectors that cater for things like that, like Slur and Tie. It at least used to in 2.3.2, seem this broke sometume in 3.x

Reply:
See `InspectorSlurTie()` and `InspectorCaesura()`

      bool sameTypes = true;
      for (const auto& ee : *inspector->el()) {
            if (ee->accessibleInfo() != b->accessibleInfo()) {
                  sameTypes = false;
                  break;
                  }
            }
      if (sameTypes)
            c.title->setText(b->accessibleInfo());

Reply:
This is a fairly good idea, but that second disadvantage is surely a "blocker".

Reply:
But one that needs to get sorted anyway. Maybe you find the cause?

Reply:
I'll probably do that once I get to compile again.

Reply:
I've created an issue for that: https://musescore.org/en/node/298363.